src/_data/catalog/widgets.json

```diff
@@ -70,7 +70,7 @@
       "Information displays"
     ],
     "link": "https://api.flutter.dev/flutter/widgets/Icon-class.html",
-    "image": "<img alt='' src='/images/catalog-widget-placeholder.png'>"
+    "image": "<img alt='' src='https://raw.githubusercontent.com/flutter/assets-for-api-docs/master/assets/widgets/icon.png'>"
```

Comment:
```suggestion
    "image": "<img alt='' src='https://flutter.github.io/flutter/assets-for-api-docs/master/assets/widgets/icon.png'>"
```
I would wait and double check that this resolves to the desired asset when it lands in the repo. :)

Reply:
ingore previous. This is the proper link:
https://flutter.github.io/assets-for-api-docs/assets/widgets/icon.png

Reply:
Thanks!